addons/html_builder/static/src/core/field_change_replication_plugin.js

```diff
@@ -0,0 +1,128 @@
+import { Plugin } from "@html_editor/plugin";
+import { closestElement, selectElements } from "@html_editor/utils/dom_traversal";
+import { withSequence } from "@html_editor/utils/resource";
+
+export class FieldChangeReplicationPlugin extends Plugin {
+    static id = "fieldChangeReplication";
+
+    resources = {
+        handleNewRecords: this.handleMutations.bind(this),
```

Comment:
I don t know if you see the change. handleMutations it s only trigger on user mutation. We don t trigger it for normalize mutation